scripts/get_version.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
 # Get VERSION from settings.py
-grep -Eo -m 1 'VERSION\s+= .*(\d+)\.\d+\.\d+.*' main/settings.py |
```

Comment:
`\d` seems not to work on docker / github actions

Reply:
\d isn't supported in bash. You can use [:digit:] if you want, but what you have is just fine :)

[Cite](https://stackoverflow.com/questions/17968020/how-to-match-digits-in-regex-in-bash-script)

Reply:
Thanks! Good to know.